databricks/resource_databricks_azure_blob_mount_test.go

```diff
@@ -127,38 +127,19 @@ func testAccAzureBlobMountMountExists(n string, azureBlobMount *AzureBlobMount,
 }
 
 func testAccAzureBlobMountCorrectlyMounts() string {
-	clientID := os.Getenv("ARM_CLIENT_ID")
-	clientSecret := os.Getenv("ARM_CLIENT_SECRET")
-	tenantID := os.Getenv("ARM_TENANT_ID")
-	subscriptionID := os.Getenv("ARM_SUBSCRIPTION_ID")
-	workspaceName := os.Getenv("TEST_WORKSPACE_NAME")
-	resourceGroupName := os.Getenv("TEST_RESOURCE_GROUP")
-	managedResourceGroupName := os.Getenv("TEST_MANAGED_RESOURCE_GROUP")
-	location := os.Getenv("TEST_LOCATION")
 	blobAccountKey := os.Getenv("TEST_STORAGE_ACCOUNT_KEY")
 	blobAccountName := os.Getenv("TEST_STORAGE_ACCOUNT_NAME")
 
 	definition := fmt.Sprintf(`
-	provider "databricks" {
-	  azure_auth = {
-		client_id              = "%[1]s"
-		client_secret          = "%[2]s"
-		tenant_id              = "%[3]s"
-		subscription_id        = "%[4]s"
-
-		workspace_name         = "%[5]s"
-		resource_group         = "%[6]s"
-		managed_resource_group = "%[7]s"
-		azure_region           = "%[8]s"
-	  }
-	}
-
```

Comment:
Hey @EliiseS i am concerned that removing this block will not allow the environment variables to propagate to the databricks provider azure_auth block for SP based auth. Since this is not defined by default GetOk("azure_auth") will be skipped please let me know if I am missing something.

Reply:
```make`
terraform-acc-azure: lint
	@echo "==> Running Terraform Acceptance Tests for Azure..."
	@/bin/bash integration-environment-azure/run.sh
```

The `run.sh` script called in the make file sets up the environment variables

Reply:
Yeah we unified the approach as the other Azure integration tests relied on the `init` block in `provider_test.go` to populated the provider. We update the `run.sh` for Azure integration tests to output the right Environment vars and now the mount tests can use the same path and omit the `provider` block in their tests.

![image](https://user-images.githubusercontent.com/1939288/85275503-55655180-b478-11ea-874c-4c104f3a21d6.png)

Reply:
The weird part about this is if you are using maps the environment variables do not populate correctly unless azure_auth section exists. I am confused as to how this integration test is working. Only token and host env variables populate correctly.

Reply:
In the [`init` section of `provider_test.go` here](https://github.com/databrickslabs/terraform-provider-databricks/blob/c343db6a0c03b98d0f038b8a6c628d3ce2962db9/databricks/provider_test.go#L31) the `providerConfigureAzureClient` is called programmatically and it [creates a new map here](https://github.com/databrickslabs/terraform-provider-databricks/blob/c343db6a0c03b98d0f038b8a6c628d3ce2962db9/databricks/provider.go#L173) then assigns to if from the environment variables. (as you say without this during a normal tf init the defaults wouldn't be applied). 

Make sense? Took me a while to work out but it's much neater than having the provider initialized in each test.

Reply:
We've figured out that the providerConfigureAzureClient sets the azure_auth